composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": "^5.3 || ^7.0"
+        "php": "^5.3 || ^7.0 || ^8.0"
```

Comment:
Ha, that was easy! Is it all?

I probably should update the CI infra too...

Reply:
for the CI, this will require updating to newer PHPUnit versions. The old PHPUnit 5.7 does not support PHP 8

Reply:
Yeah, I suppose we can use the symfony phpunit "bridge" (or whatever it's called), right?

Reply:
Well, using composer is also fine.

But in both cases, you will need to adapt the testsuite to be compatible with newer PHPUnit versions. Note that dropping support for older PHP versions might help a bit here, by reducing the range of PHPUnit versions you need to support. Supporting PHPUnit 4 and 8 at the same time is quite painful. See https://phpunit.de/supported-versions.html for the supported PHP versions for each PHPUnit major version.